src/components/pagination/pagination.scss

```diff
@@ -245,6 +245,7 @@ $indicator-width-current: $indicator-height-current;
  * focus ring similar to other components to the number element.
  */
 
+// stylelint-disable-next-line no-duplicate-selectors
```

Comment:
`no-duplicate-selectors` is a good rule, but in this case I wasn't sure if reformatting this into a parent selector would break the mixin, so I opted to ignore it.